Kbs.Wpf/User/Read/Index/ReadIndexUserUserViewModel.cs

```diff
@@ -31,7 +32,14 @@ public string Role
         public ReadIndexUserUserViewModel(UserEntity user)
         {
             ThrowHelper.ThrowIfNull(user);
-            Name = user.Name;
+            if (user.Name.IsNullOrEmpty())
```

Comment:
Gebruik string.IsNullOrEmpty(...)
```suggestion
            if (string.IsNullOrEmpty(user.Name))
```